modules/yieldlabBidAdapter.js

```diff
@@ -81,8 +81,8 @@ export const spec = {
         const bidResponse = {
           requestId: bidRequest.bidId,
           cpm: matchedBid.price / 100,
-          width: customsize[0],
-          height: customsize[1],
+          width: bidRequest.params.useSlotSize !== undefined ? primarysize[0] : customsize[0],
```

Comment:
I don't see the `useSlotSize` parameter in your documentation -- http://prebid.org/dev-docs/bidders.html#yieldlab

Is this a new param? If so, please submit a PR for the docs site. https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/yieldlab.md

Reply:
Hi @bretg 
I've changed the documentation: https://github.com/prebid/prebid.github.io/pull/1294